aws-lambda-layer/README.md

```diff
@@ -0,0 +1,44 @@
+# FreeTDS Lambda Layer
+
+Build and publish FreeTDS as an AWS Lambda layer.
+
+In order to use the TinyTDS gem with ruby AWS Lambda Functions, we need to have access to FreeTDS binaries that AWS does not provide with its execution environment.
+
+The solution is to use AWS Lambda Layers.
+
+The provided deploy script builds FreeTDS on the same architecture that runs in AWS Lambda and, using the AWS CLI, publishes the artifact as an AWS Lambda layer for functions to use.
+
+> A Lambda deployment package is a .zip file that contains your code and any dependencies. The Lambda execution environment is based on a specific Amazon Linux AMI and kernel version. Any native binaries that are used in a Lambda deployment package must be compiled in this environment, and only 64-bit binaries are supported.
+> 
+> https://aws.amazon.com/premiumsupport/knowledge-center/lambda-linux-binary-package/
+
+## Requirements
+
+- Ruby 2.5.3
```

Comment:
Is this rake task really that specific?

Reply:
No, I guess not but you do need to ruby to run the rake tasks and the layer is built on a container using ruby2.5 so I figure I would add 

This line could just be "Ruby 2.5.x" or just "Ruby".